common/src/main/java/com/zimbra/common/localconfig/LC.java

```diff
@@ -1470,7 +1470,7 @@ public enum PUBLIC_SHARE_VISIBILITY {
   @Retention(RetentionPolicy.RUNTIME)
   public @interface Supported {}
 
-  public static PUBLIC_SHARE_VISIBILITY getPublicShareAdvertisingScope() {
+  public static_SHARE_VISIBILITY getPublicShareAdvertisingScope() {
```

Comment:
Thid didn't worked :smiling_face_with_tear: 

```suggestion
  public static PUBLIC_SHARE_VISIBILITY getPublicShareAdvertisingScope() {
```